numpy/core/src/npymath/ieee754.c.src

```diff
@@ -204,7 +204,6 @@ static npy_longdouble _nextl(npy_longdouble x, int p)
         }
     }
 
-    npy_longdouble u;
```

Comment:
This is an extra change, gcc on my ppc system just started complaining about this declaration.

Reply:
If you are running `runtests.py` to test and have rebased off master, we recently extended the `-Werror=*` flags, this might be the source.

Reply:
I don't think this code path was being compiled before - I'd expect an error due to C90 declarations after statements.

Reply:
Are you able to build `HEAD^` with `bool obvious_syntax_error = true false;` on this line? (ie, was it being compiled before the macro changes?

Reply:
I just tried compiling master, it fails too in the same way.

So I'm 99% sure this is the new warning setting.

Reply:
Also, I can't actually test the macro changes. I have a ppc system, but it's not a darwin system.  So I was definitely compiling this code path before.

Reply:
yes, I am running runtests.py